src/Microsoft.Azure.WebJobs.Protocols/JsonSerialization.cs

```diff
@@ -85,6 +85,17 @@ internal static JsonTextWriter CreateJsonTextWriter(TextWriter textWriter)
             return jsonWriter;
         }
 
+        public static bool IsValidJObjectInput(string input)
```

Comment:
nit: recommend renaming this to IsJsonObject - we're not really checking whether it's valid json, so "valid" is a little strong